py_ocpi/commands/v_2_2_1/api/cpo.py

```diff
@@ -1,10 +1,50 @@
-from fastapi import APIRouter
+from typing import Union
+
+from fastapi import APIRouter, Depends
+from pydantic import ValidationError
+
+from py_ocpi.core.dependencies import get_crud, get_adapter
+from py_ocpi.core.enums import ModuleID
+from py_ocpi.core.schemas import OCPIResponse
+from py_ocpi.core import status
+from py_ocpi.commands.v_2_2_1.enums import CommandType
+from py_ocpi.commands.v_2_2_1.schemas import (CancelReservation, ReserveNow, StartSession, StopSession, UnlockConnector,
+                                              CommandResponse, CommandResult)
 
 router = APIRouter(
     prefix='/commands',
 )
 
 
-@router.get("/")
-def get_commands():
-    return "commands app created!"
+@router.post("/{command}", response_model=OCPIResponse)
+async def send_commands_receiver_interface(
+        command: CommandType,
+        data: Union[CancelReservation, ReserveNow, StartSession, StopSession, UnlockConnector],
+        crud=Depends(get_crud), adapter=Depends(get_adapter)):
+    try:
+        response = await crud.create(ModuleID.commands, dict(**data.dict(), command=command))
```

Comment:
it's better to move `command=command` out of dict and make it a parameter for crud.create